### DISCUSSION TITLE
Is it possible to also replace calls to require() function ?

### DISCUSSION THREAD

**Discussion by meypod** [justkey007/tsc-alias#184] [Q&A]
Hi
I'm trying to use this in a react native mono repo
I wanted to know if it's possible to also replace the paths inside the `require()` calls that exists in the project ?
I am using require() in my ts files, which will be later picked up by babel, but since it's in another package, the require paths resolve to wrong locations, which I need to tackle on my build step of the package

it seems to me if I manage to make tsc-alias to also work for `require()`, my problem will be solved

**Comment by justkey007** [comment]
I advise you to try it and see the result yourself and in case it doesn't work and seems like a bug you can create an issue by leaving a repo there for its reproduction. THANKS

**Reply by meypod** [reply]
I tried, by manually replacing all with relative paths, then I realized that since I use it for assets, the assets then needs to be copied over Or the path should point to the correct files in the src folder
So overall I think I'm taking a wrong approach here
